canonical-web-and-design/ubuntu.com
# Update vm cta on /appliance/<appliance> page

## Done

- Update vm cta on /appliance/<appliance> page
- Made the description block 9 by 3 cols
- Removed the 'Build your own appliance' strip

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/plex
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1zrIMPpiSLxNp44vMFundmV3jZ4Df_eilGdJm0SUlUHo/edit)
